lib/src/screenshots.dart

```diff
@@ -292,37 +319,71 @@ Future<List<String>> _generateThumbnails(String originalPath, String webpPath,
   final width = int.parse(widthString.split(':').last.trim());
   final height = int.parse(heightString.split(':').last.trim());
 
-  final int widthArgument;
-  final int heightArgument;
+  final int width100Argument;
+  final int height100Argument;
+  final int width190Argument;
+  final int height190Argument;
+
   if (width > height) {
-    widthArgument = 100;
-    heightArgument = 0;
+    width100Argument = 100;
+    width190Argument = 190;
+    height100Argument = 0;
+    height190Argument = 0;
   } else {
-    widthArgument = 0;
-    heightArgument = 100;
+    width100Argument = 0;
+    width190Argument = 0;
+    height100Argument = 100;
+    height190Argument = 190;
   }
```

Comment:
Perhaps refactor this to a helper function

Reply:
Done